.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+ - 2.6.4
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database junk4dollars_test;' -U postgres
+script:
+  - bundle exec rake test
```

Comment:
Should ci run rubocop too?

Reply:
Yep! forgot about that. let me get that added